src/quantuminspire/qiskit/backend_qx.py

```diff
@@ -286,18 +284,17 @@ def _collect_measurements(experiment):
         return {'measurements': measurements, 'number_of_clbits': number_of_clbits}
 
     @staticmethod
-    def __qubit_to_classical_hex(qubit_register, measurements, number_of_qubits):
+    def __qubit_to_classical_hex(qubit_register: int, measurements: Dict, number_of_qubits: int) -> str:
```

Comment:
Do we know more about measurements type, other than it's just a dict?

Reply:
Added type hint for Dict